docs/cmdline-opts/disable.d

```diff
@@ -9,6 +9,9 @@ Added: 5.0
 See-also: config
 Multi: boolean
 ---
-If used as the first parameter on the command line, the *curlrc* config
+If used as the **first** parameter on the command line, the *curlrc* config
 file will not be read and used. See the --config for details on the default
 config file search path.
+
+Prior to 7.50.0 curl supported the short option name *-q* but not the long
+option name *--disable*.
```

Comment:
Maybe you can just avoid naming the long option in this last paragraph?

Reply:
I removed the dashes so it should show as this now:
Prior to 7.50.0 curl supported the short option name *q* but not the long option name *disable*.